### PR TITLE
0.25 upgrade

### PR DESCRIPTION
**Pull request by lizconlan**
Connects to #148

### CODE REVIEW

**Comment by lizconlan** [comment]
Haven't got the look quite the same - the logo has moved by about 1px and the banner section takes up slightly less room, which... might be good? A second opinion on that would be great!

**Comment by garethrees** [comment]
> Haven't got the look quite the same

Screenshot plz! :)

File: lib/views/user/_signin.html.erb [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-<%= form_tag({:action => "signin"}, {:id => "signin_form"}) do %>
-    <%= foi_error_messages_for :user_signin %>
-
-    <!--<% if not sign_in_as_existing_user %>
-    <h2><%= _('If you\'ve used {{site_name}} before', :site_name=>site_name)%></h2>
-    <% end %>-->
-
-    <p>
-    <label class="form_label" for="user_signin_email"><%= _('Your e-mail:')%></label>
-    <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 120 } %>
```
Comment (garethrees) [inline lib/views/user/_signin.html.erb]: Looks different to me:

```diff
- <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 120 } %>
+ <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 60 } %>
```

https://github.com/mysociety/alaveteli/blob/develop/app/views/user/_signin.html.erb#L7

See https://github.com/mysociety/derechoapreguntar-theme/commit/458ad320a0b9dd9bdf3d90cdc0bb05529b519606#diff-0c4b67a5a78077d4a261901b2022850aR11

File: lib/views/user/_signup.html.erb [hunk 2]
```diff
@@ -63,15 +64,13 @@
...
     <% if @request_from_foreign_country %>
-    <%= recaptcha_tags %>
+      <%= recaptcha_tags %>
     <% end %>
 
     <div class="form_button">
       <%= hidden_field_tag 'token', params[:token], { :id => 'signup_token' } %>
       <%= hidden_field_tag :modal, params[:modal], { :id => 'signup_modal' } %>
-      <%= f.submit _('Sign up'), :tabindex => 110 %>
+      <%= f.submit _('Sign up'), :tabindex => 50, :disable_with => _("Sending...") %>
```
Comment (garethrees) [inline lib/views/user/_signup.html.erb]: Probably want to keep the tabindex?

**Comment by garethrees** [comment]
Footer should have `role="contentinfo"`

File: lib/views/user/_show_user_info.html.erb [hunk 3]
```diff
@@ -1,10 +1,10 @@
-<% if !@display_user.get_about_me_for_html_display.empty? || @is_you %>
+<% if (!@display_user.banned? && !@display_user.get_about_me_for_html_display.empty?) || @is_you %>
   <div class="user_about_me">
     <%= image_tag "navimg/quote-open.png", :class => "comment_quote" %>
```
Comment (garethrees) [inline lib/views/user/_show_user_info.html.erb]: The image path is different in core

**Comment by garethrees** [comment]
Looks good on the whole. Couple of missed bits. Have you checked the Ruby overrides? There are quite a few and not sure if any conflict.

**Comment by lizconlan** [comment]
The overrides don't look too controversial - mostly extra `User` fields and a custom state for extended deadlines. Oh and hacking in the Public Body categories for `es_NI`

**Comment by lizconlan** [comment]
## Homepage
nicaragua-screenshot

## Authority page
nicaragua-authority

## Request page
nicaragua-request

## Mobile view
nicaragua-mobile


## Mobile menu detail
nicaragua-mobile-menu

**Comment by garethrees** [comment]
Looks good. Squash it down but we'll have to wait 'til translations are done before merge.

Also need to generate the theme translations, but will do that after merge.

**Pull request closed without merging**